mobile/native/src/ln_dlc/mod.rs

```diff
@@ -544,13 +537,23 @@ fn derive_trades_from_filled_orders() -> Result<Vec<WalletHistoryItem>> {
                 .trader_margin()
                 .expect("Filled order to have a margin");
 
+            let execution_price = Decimal::try_from(
+                first
+                    .execution_price()
+                    .context("execution price to be set on a filled order")?,
```

Comment:
🙃 I think you want an `expect` here. Your choice of words seems to indicate that.

Reply:
you are competing with the review of @bonomat 🙈 https://github.com/get10101/10101/pull/1497#discussion_r1365099744

but maybe I've miss-interpreted his remark?